cannot build ncspot with cargo

```
   Compiling librespot-protocol v0.1.1
error[E0308]: mismatched types
  --> /home/davidemaglio/.cargo/registry/src/github.com-1ecc6299db9ec823/librespot-protocol-0.1.1/build.rs:45:39
   |
45 |         let p = parse_and_typecheck(&["proto"], &[src]).expect("protoc");
   |                                       ^^^^^^^ expected struct `std::path::Path`, found `str`
   |
   = note: expected reference `&std::path::Path`
              found reference `&'static str`

error[E0308]: mismatched types
  --> /home/davidemaglio/.cargo/registry/src/github.com-1ecc6299db9ec823/librespot-protocol-0.1.1/build.rs:45:51
   |
45 |         let p = parse_and_typecheck(&["proto"], &[src]).expect("protoc");
   |                                                   ^^^ expected struct `std::path::Path`, found struct `std::string::String`
   |
   = note: expected reference `&std::path::Path`
              found reference `&std::string::String`

error: aborting due to 2 previous errors

For more information about this error, try `rustc --explain E0308`.
error: could not compile `librespot-protocol`.
warning: build failed, waiting for other jobs to finish...
error: failed to compile `ncspot v0.1.3`, intermediate artifacts can be found at `/tmp/cargo-installbsLCky`

Caused by:
  build failed

```

Same here.

	$ cargo install librespot
	...
	   Compiling librespot-protocol v0.1.1
	error[E0308]: mismatched types
	  --> /home/me/.cargo/registry/src/github.com-1ecc6299db9ec823/librespot-protocol-0.1.1/build.rs:45:39
	   |
	45 |         let p = parse_and_typecheck(&["proto"], &[src]).expect("protoc");
	   |                                       ^^^^^^^ expected struct `std::path::Path`, found `str`
	   |
	   = note: expected reference `&std::path::Path`
		          found reference `&'static str`

	error[E0308]: mismatched types
	  --> /home/me/.cargo/registry/src/github.com-1ecc6299db9ec823/librespot-protocol-0.1.1/build.rs:45:51
	   |
	45 |         let p = parse_and_typecheck(&["proto"], &[src]).expect("protoc");
	   |                                                   ^^^ expected struct `std::path::Path`, found struct `std::string::String`
	   |
	   = note: expected reference `&std::path::Path`
		          found reference `&std::string::String`

	error: aborting due to 2 previous errors

	For more information about this error, try `rustc --explain E0308`.
	error: could not compile `librespot-protocol`.
	warning: build failed, waiting for other jobs to finish...
	error: failed to compile `librespot v0.1.1`, intermediate artifacts can be found at `/tmp/cargo-installhkxtbM`

	Caused by:
	  build failed

---

	$ rustc --version
	rustc 1.42.0 (b8cedc004 2020-03-09)

---

	$ swupd info 
	Distribution:      Clear Linux OS
	Installed version: 32840

--- a/protocol/Cargo.toml
+++ b/protocol/Cargo.toml
@@ -8,8 +8,8 @@ license="MIT"
 edition = "2018"
 
 [dependencies]
-protobuf = "2.8.1"
+protobuf = "~2.8.1"
 
 [build-dependencies]
-protobuf-codegen-pure = "2.8.1"
-protobuf-codegen = "2.8.1"
+protobuf-codegen-pure = "~2.8.1"
+protobuf-codegen = "~2.8.1"

